l10n_be_cooperator_national_number/models/subscription_request.py

```diff
@@ -0,0 +1,70 @@
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class SubscriptionRequest(models.Model):
+    _inherit = "subscription.request"
+
+    national_number = fields.Char(string="National Number")
+    display_national_number = fields.Boolean(
+        compute="_compute_display_national_number",
+    )
+
+    @api.depends("company_id", "company_id.require_national_number")
+    def _compute_display_national_number(self):
+        self.display_national_number = self.company_id.require_national_number
```

Comment:
This seems really weird to me. Should it not also be displayed if `require_national_number` is False? Shouldn't this field be set to False if `is_company` is True?

Reply:
It's a company-specific settings. You can configure, by company, if you want to display the national number form.

Reply:
I'm implying there's a difference between _requiring_ a national number and _displaying_ the national number field.

This code assumes that 'require == display', and 'not require == not display'. I think that's a weird design. I'll open an issue for that to expedite this PR.